app/lib/onboarding/v2_onboarding_categories_test.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Tech
+  items:
+    - account: '@foo@bar.baz'
```

Comment:
It would be good if these were actual accounts (we will try to show their profile if the user taps on them). [these seem fake?]